index.js

```diff
@@ -41,7 +68,19 @@ app.get("/movies/:Title", (req, res) => {
 
 //Get list of movies by director
 app.get("/movies/Director/:Director", (req, res) => {
-  Movies.findOne({Director: req.params.Director})
+  Movies.find({'Director.Name': req.params.Director})
```

Comment:
Since this endpoint should be returning a list of movies, I recommend using the method `find` instead of `findOne` so it returns all the movies that match the criteria